examples/analog_hamiltonian_simulation/01_Introduction_to_Aquila.ipynb

```diff
@@ -7,175 +7,203 @@
     "tags": []
```

Comment:
<p>1) Let's keep this simple for now (per our agreement with QuEra, all shots are successful at the moment) "<code>status</code> indicates if the shot is successful, or failed or partially successful" -&gt; " <code>status</code> indicates if the shot is successful"</p><p>2) To point 2., Let's add a note (Can we make it a blue-colored box maybe?): "Some atoms may be missing even if the shot was successful. We recommend comparing <code>pre_sequence</code> of each shot with the requested atom filling in the AHS program specification."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>